serveradmin/powerdns/utils.py

```diff
@@ -13,7 +13,12 @@ def get_domain_settings() -> List:
     Returns the PDNS.domain django settings or an empty list if not setup.
     """
     if settings.PDNS:
+<<<<<<< HEAD
```

Comment:
You somehow ended up bringing in merge conflict diffs. I guess that was not intended. ^^

Reply:
No I must have missed that. Removed it.